nvtabular/ops/categorify.py

```diff
@@ -1132,6 +1205,7 @@ def _encode(
     elif dtype:
         labels = labels.astype(dtype, copy=False)
 
+    labels = labels + (start_index - 1)
```

Comment:
I don't think this quite works with on the GPU with list columns - can you move  this line up to before the 'dispatch._encode_list_column' call (move to line 1202).  This will do the start_index adjustment before we encode as a list column and should fix the unittest failures you're seeing here

Reply:
Intriguing, yes, let me try this update as you suggest.

Reply:
Indeed this has resolved all the failing tests related to the `Categorify()` class.

It looks like there is one failing test for `test_notebooks.py` and two for `test_s3.py`.

Would you have a suggested way that I can debug these two, if these two failing tests are related to the changes in this PR?

I have the Jenkins build information available at this URL: http://10.20.13.93:8080/job/nvtabular_tests/3390/

Reply:
Thanks Ben, it looks like the merging main with the changes in #1079 resolved this three last failing tests. I would say this current PR #1093 is ready for review and to be merged into main.